source/events/rdo-day-fosdem-2016.html.md

```diff
@@ -60,6 +60,8 @@ allowfullscreen></iframe>
 
 ## RDO configuration files: do's, dont's, gotchas, and rfc - *Ihar Hrachyshka*
 
+[[Slides](https://docs.google.com/presentation/d/1MElADyGDLx_26poFDqMRdpDV0HJkBb8drhvE2604mf4/pub)
```

Comment:
> You need permission to access this published document.

@rbowen @booxter 
This is not public, I guess it needs to be moved out of Red Hat org?